### DISCUSSION TITLE
part-ii/2018-46/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#161] [Question comments]
Part II 2018 Differential Geometry: Paper 4, Section II, I 

https://questions.tripos.org/part-ii/2018-46/

**Comment by stevezshen** [comment]
<details>
<summary>Spoiler warning</summary>

c)i) Consider a point p on the boundary $\partial S\subset \tilde{S}$. The Gaussian curvature is defined and smooth in a neighbourhood of $p$. Consider a geodesic $\gamma$ with $\gamma'(0)$ the $d\phi$ image of a vector lying in the half plane corresponding to $S$. This is a maximal geodesic which terminates in finite time, so the curvature should blow up in a nhood of $p$, contradiction.
(ii) a pinched cone. (the surface of revolution of $\sqrt{x}$)

Feedback appreciated

</details>

**Comment by ArthurConmy** [comment]
<details>
<summary>Final part</summary>

The cone $x^2 + y^2 = z^2, z > 0$ works.

</details>
